tools/Makefile

```diff
@@ -160,13 +174,22 @@ ifneq ($(CONFIG_CCACHE)$(CONFIG_SDK),)
   tools-noccache :=
   tools-noccache += autoconf
   tools-noccache += automake
+  tools-noccache += bison
   tools-noccache += cmake
+  tools-noccache += e2fsprogs
```

Comment:
why is this here?

Reply:
cmake --> ninja --> python3 --> e2fsprogs --> util-linux --> bison --> flex --> libtool

Reply:
it may be possible to eliminate e2fsprogs and just depend on util-linux but it doesn't hurt this way

Reply:
right the original reason python3 depended on e2fsprogs was for its libuuid. I replaced that with util-linux.

I think tools/e2fsprogs is mainly here for some tools used for generating firmware images.

Reply:
replaced e2fs dependency with util-linux to be more clear